#include "IdAllocatorServiceImpl.hpp"
int IdAllocatorServiceImpl::AllocateId(const ::idallocator::AllocateIdReq & oReq, ::idallocator::AllocateIdResp & oResp) {
    return -1;
}
int IdAllocatorServiceImpl::BatchAllocateId(const ::idallocator::BatchAllocateIdReq & oReq, ::idallocator::BatchAllocateIdResp & oResp) {
    return -1;
}
